src/Symfony/Bundle/TwigBundle/DependencyInjection/Compiler/ExtensionPass.php

```diff
@@ -79,7 +79,7 @@ public function process(ContainerBuilder $container)
             $container->removeDefinition('twig.controller.preview_error');
         }
 
-        if ($container->has('request_stack')) {
+        if ($container->has('router.url_helper')) {
```

Comment:
@vudaltsov as discussed I think this change can cause some problems for apps that will only update TwigBundle?